spec/file_reference_installer/copy_xcassets_file_referenc_installer_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+
+#TODO: Figure out shorter class name
+module Xcake
+  describe CopyXCAssetsFileReferenceInstaller do
+    before :each do
+      @node = double("Node").as_null_object
+    end
+
+    it "should not able to install folder" do
+      folder_path = 'Folder/'
+
+      allow(@node).to receive(:path).and_return(folder_path)
+      allow(File).to receive(:directory?).with(folder_path).and_return(true)
+
+      expect(CopyXCAssetsFileReferenceInstaller.can_install_node(@node)).to be(false)
+    end
+
+    it "should be able to install xcassets foler" do
+      folder_path = 'Assets.xcassets'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.